scrapers/koji_scraper.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-License-Identifier: GPL-3.0+
+from scrapers.base import BaseScraper
+from purview.models.koji import KojiBuild, KojiTask
+from purview.models.user import User
+from datetime import datetime
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class KojiScraper(BaseScraper):
+
+    def run(self, since=None):
+        log.info('Connected to TEIID, fetching initial load for koji')
+        # Initialize a start date from which all builds must be fetched
+        # Currently hardcoding this to fetch builds from the past two years
+        if since is None:
+            since = datetime(2016, 3, 1)
+        self.builds = self.get_initial_load_of_koji_builds(since)
+        self.upload_data_to_neo4j(self.builds)
+        log.info('Initial load for Koji builds Complete!')
+
+    def get_initial_load_of_koji_builds(self, start_date):
+        # SQL query to fetch all builds from start date until now
+        log.info('Getting all Koji builds since {0} koji builds'.format(start_date))
+        self.sql_query = """
+                SELECT build.*, events.time as creation_time, package.name as package_name
+                FROM build
+                LEFT JOIN events
+                ON build.create_event = events.id
+                LEFT JOIN package
+                ON build.pkg_id = package.id
+                WHERE build.completion_time IS NOT NULL AND build.completion_time > '{}'
+                """.format(start_date)
+
+        # Getting builds from teiid
```

Comment:
Bike-shedding: This comment isn't very useful since it's obvious that's happening below. If you want to keep it, then make it a debug statement.